src/abstractdataframe/abstractdataframe.jl

```diff
@@ -1438,6 +1438,52 @@ julia> unique(df, 2)
 ─────┼──────────────
    1 │     1      1
    2 │     2      2
+```
+"""
+unique
```

Comment:
Attach this to the actual method now that it's possible?

Reply:
done